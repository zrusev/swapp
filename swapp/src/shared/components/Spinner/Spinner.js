import React from 'react';
import { OrbitSpinner } from 'react-epic-spinners';
import { withTheme } from 'styled-components';

export default withTheme(({ theme: { primaryHeadingFontColor } }) => (
  <OrbitSpinner
    size={150}
    color={primaryHeadingFontColor}
    style={{ margin: '10em auto' }}
  />
));
