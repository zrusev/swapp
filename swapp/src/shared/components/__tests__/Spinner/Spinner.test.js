import React from 'react';
import { shallow } from 'enzyme';

import Spinner from '../../Spinner/Spinner';

describe('should mount spinner', () => {
  it('should contain spinner', () => {
    const props = {
      theme: {
        primaryHeadingFontColor: 'yellow',
      },
    };

    const wrapper = shallow(<Spinner {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
