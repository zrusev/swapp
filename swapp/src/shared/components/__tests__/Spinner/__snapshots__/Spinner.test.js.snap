// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should mount spinner should contain spinner 1`] = `
<Component
  theme={
    Object {
      "primaryHeadingFontColor": "yellow",
    }
  }
/>
`;
